Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -1624,9 +1624,6 @@ void RemoveRefresh()
 
 		void UpdateContentOffset(nfloat offset, Action completed = null)
 		{
-			if (!_usingLargeTitles)
```

Comment:
Why remove this?

Reply:
The fix calls this method and we don't want it to only happen when using large titles. Nothing bad happens if you call this for an app without large titles, the offset is still correctly calculated and in fact the animation looks better. I can show in gif attachment tomorrow.